apps/test/unit/code-studio/scriptEditorReduxTest.js

```diff
@@ -65,6 +68,16 @@ describe('scriptEditorRedux reducer tests', () => {
     );
   });
 
+  it('mapLessonGroupDataForEditor', () => {
```

Comment:
I wrote this test very quickly and I am definitely open to suggestions to improve it

Reply:
I might check fields like the name as well but overall this seems good.